README.md

```diff
@@ -0,0 +1,26 @@
+# Django MongoDB Project Template
+
+This is the starter template for the Django-MongoDB Backend. In order to use, with your version of `django-mongodb-backend` and `django`:
+
+* Find your Django version. To do so from the command line, make sure you have django installed and type:
+
+```bash
+django-admin --version
+>> 5.0
+```
+
+
+## Create the Django Project
+From your shell, run the following command to create a new Django project replacing the `{{ project_name }}` and `{{ version }}` sections. 
+
+```bash
+django-admin startproject {{ project_name }} --template https://github.com/mongodb-labs/django-mongodb-project/archive/refs/heads/{{ version }}.x.zip
```

Comment:
Probably would prefer django-admin startproject mysite --template https://github.com/mongodb-labs/django-mongodb-project/archive/refs/heads/main.zip (or 5.0.x.zip if we create branches).